### DISCUSSION TITLE
Auto ID in link

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#7302] [Help] [answered]
Hey,

I have a problem with one of my apps.
Im trying to redirect the user with the Auto ID from the row to the .../:id screen. 
It works in one of my apps but it won't work in my newer app.
The other auto columns would work but I don't want to use them because they're hard to read and not pleasant to look at.

Please help me

**Comment by R2bEEaton** [comment]
Hi @kasgithub8!

In what way does it not work? Could you provide more information of what you've tried and potentially an export.

**Reply by kasgithub8** [reply]
Gotcha, im gonna send in the export tomorrow.

**Reply by kasgithub8** [reply]
@R2bEEaton,

what was the email for the exports again?

**Reply by melohagan** [reply]
Hi @kasgithub8 Could you send your app export to support@budibase.com and I'll have a look

**Reply by kasgithub8** [reply]
> Hi @kasgithub8 Could you send your app export to [support@budibase.com](mailto:support@budibase.com) and I'll have a look

Just did. Don't get confused tho, I'm using _id for links rn.

**Comment by melohagan** [comment, marked as answer]
Hi @kasgithub8 

I was able to get this to work with the following:

**Navigate To**: `/requirement-requests-power/{{ RequirementRequests Table.RequirementRequests.Auto ID }}`

**Data provider filter**
![Screenshot 2022-08-18 at 14 41 27](https://user-images.githubusercontent.com/101575380/185409604-3e261d43-801b-4e58-879f-2bf3434c0707.png)
![Screenshot 2022-08-18 at 14 41 46](https://user-images.githubusercontent.com/101575380/185409674-f2801a19-6b78-41f0-8c27-195994012304.png)

**Reply by kasgithub8** [reply]
@melohagan omg. I didn't even know that this binding existed. Thank you so much! 😄
